requirements-dev.txt

```diff
@@ -1,12 +1,12 @@
 -r requirements.txt
-flake8==3.9.0
-ipython==8.10.0
-pytest==8.0.2
-pytest-django==4.2.0
-pytest-env==0.6.2
-pytest-cov==3.0.0
-pytest-xdist[psutil]==2.5.0
-mkdocs-material==8.5.11
-mkdocs-redirects==1.2.0
-responses==0.22.0
-SQLAlchemy-Utils==0.38.2
+flake8==7.1.1
+ipython==7.34.0
```

Comment:
I guess it doesn't really matter, but why did this get downgraded?

Reply:
Oh, that was an accident, reverted in https://github.com/mathesar-foundation/mathesar/pull/3990.